docs/features/Docker Support.md

```diff
@@ -1,7 +1,33 @@
 Quartz comes shipped with a Docker image that will allow you to preview your Quartz locally without installing Node.
 
-You can run the below one-liner to run Quartz in Docker.
+You can run Quartz in Docker. A container image is available at:
+
+- `ghcr.io/jackyzha0/quartz`
+
+You can create a local build with:
+
+```sh
+docker build --load -t ghcr.io/jackyzha0/quartz .
+```
+
+This example expects a `content/` directory with the `quartz.*.ts` configuration files and a (empty) `public/` directory to be present in the preesent working directory.
+
+```sh
+mkdir -p content/ public/
+```
+
+Build the site with running a local development web server for preview:
+
+```sh
+docker run --rm -it -p 8080:8080 -v ${PWD}/content:/usr/src/app/content -v ${PWD}/public:/usr/src/app/public --entrypoint sh ghcr.io/jackyzha0/quartz -c "cp content/quartz.*.ts .; npx quartz build --serve"
```

Comment:
note that this is *nix style and wouldn't work on windows as they dont have `/usr`

Reply:
Everything to the right side of the colons `:` in the `-v` volumes definitions happens inside the container, where `/usr` is present.

Reply:
ah misread, disregard